src/components/Table/GridTable.tsx

```diff
@@ -682,7 +717,12 @@ const VirtualRoot = memoizeOne<
  * Calculates column widths using a flexible `calc()` definition that allows for consistent column alignment without the use of `<table />`, CSS Grid, etc layouts.
  * Enforces only fixed-sized units (% and px)
  */
-export function calcColumnSizes(columns: GridColumn<any>[], firstLastColumnWidth: number | undefined): string[] {
+export function calcColumnSizes(
+  columns: GridColumn<any>[],
+  firstLastColumnWidth: number | undefined,
+  tableWidth?: number,
```

Comment:
Small preference for `tableWidth: number | undefined` since I think we do want callers to always pass `tableWidth`, it just may be `undefined`